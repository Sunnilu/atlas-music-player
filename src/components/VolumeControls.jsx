// src/components/VolumeControls.jsx
export default function VolumeControls() {
  return (
    <div className="mb-6">
      <input 
        type="range" 
        className="w-full h-2 bg-gray-200 rounded-lg appearance-none cursor-pointer"
        min="0" 
        max="100"
      />
    </div>
  );
}
