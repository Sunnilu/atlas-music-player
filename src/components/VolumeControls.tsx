// src/components/VolumeControls.jsx
export default function VolumeControls() {
  return (
    <div className="mb-6">
      <input
        type="range"
        className="h-2 w-full cursor-pointer appearance-none rounded-lg bg-gray-200"
        min="0"
        max="100"
      />
    </div>
  );
}
