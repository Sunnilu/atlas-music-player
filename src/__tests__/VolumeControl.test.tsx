// src/__tests__/VolumeControl.test.tsx
import { describe, it, expect } from 'vitest';
import renderer from 'react-test-renderer';
import VolumeControls from '../components/VolumeControls';

describe('VolumeControls Snapshot', () => {
  it('renders the volume slider with default configuration', () => {
    const tree = renderer.create(<VolumeControls />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
