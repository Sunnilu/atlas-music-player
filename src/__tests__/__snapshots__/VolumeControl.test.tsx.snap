// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`VolumeControls Snapshot > renders the volume slider with default configuration 1`] = `null`;
